src/getHumanAge.test.js

```diff
@@ -1,10 +1,89 @@
 'use strict';
 
-describe('getHumanAge', () => {
-  const { getHumanAge } = require('./getHumanAge');
+const { getHumanAge } = require('./getHumanAge');
 
+describe('getHumanAge', () => {
   test('should be declared', () => {
-    expect(getHumanAge)
-      .toBeInstanceOf(Function);
+    expect(getHumanAge).toBeInstanceOf(Function);
+  });
+
+  test('return an array', () => {
+    const result = getHumanAge(15, 15);
+
+    expect(result).toBeInstanceOf(Array);
+  });
```

Comment:
no need to add this code on 10-14 lines ('return an array'). You can just remove it